molecule/default/tests/test_default.py

```diff
@@ -51,7 +84,21 @@ def test_services(host, svc):
     assert host.service(svc).is_enabled
 
 
-@pytest.mark.parametrize("command", ["docker-compose version"])
-def test_command(host, command):
+def test_commands(host):
     """Test that appropriate commands are available."""
-    assert host.run(command).rc == 0
+    distribution = host.system_info.distribution
+    codename = host.system_info.codename
+
+    if distribution in ["debian"]:
+        if codename in ["buster", "bullseye"]:
+            assert host.run("docker compose version").rc == 0
+        elif codename in ["stretch", "bookworm"]:
+            assert host.run("docker-compose version").rc == 0
+        else:
+            assert False, f"Unknown codename {codename}"
+    elif distribution in ["amzn", "fedora", "kali"]:
+        assert host.run("docker-compose version").rc == 0
+    elif distribution in ["ubuntu"]:
+        assert host.run("docker compose version").rc == 0
```

Comment:
Perhaps a comment similar to this (from this PR's description) would be helpful to add in here:

> The difference between the two Debian packages is that docker-compose provides a docker-compose command, while the docker-compose-plugin package provides docker compose functionality to the docker command.

Reply:
Please see commit 0b18fb4.